### PR TITLE
feat: add version input variable

### PR DESCRIPTION
**Pull request by M0NsTeRRR**
The purpose of this PR is to allow setting the git-cliff release version without waiting to tag a new release for this action (and also pinning to a specific one).
Unfortunately, it's not possible to update `FROM` from an argument, so we have to use bash or javaScript. This PR also addresses issue #18, as there is no Docker image to build or to pull. I preferred to download the binary to speed up the process (instead of installing python and the associated package for example).

The old action took around 6 seconds due to the Docker build, whereas this one takes 0 seconds.

If we implement this, we can close #26 as it will no longer be needed.

### CODE REVIEW

**Comment by M0NsTeRRR** [comment]
@orhun you can test in my fork with `m0nsterrr/git-cliff-action@v1.0.0`

**Comment by orhun** [comment]
Hey, thanks for the PR! The shell script is scary at first glance, can we possibly split them into different files? (e.g. `install.sh` & `run.sh`)

Also, this will require a major version bump if I'm not mistaking, right? (e.g. `v4`)

**Comment by M0NsTeRRR** [comment]
> Hey, thanks for the PR! The shell script is scary at first glance, can we possibly split them into different files? (e.g. `install.sh` & `run.sh`)
> 
> Also, this will require a major version bump if I'm not mistaking, right? (e.g. `v4`)

You're welcome. Yes, I agree...
After reading the documentation, I found that it is possible to include local GitHub Action files.
It should not be a breaking change as the user only manipulates input variables, but I'm not a GitHub Actions expert. 😊

**Comment by M0NsTeRRR** [comment]
I hate bash :smiling_face_with_tear:

**Comment by M0NsTeRRR** [comment]
You can test with `m0nsterrr/git-cliff-action@v1.3.1`

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This action generates a changelog based on your Git history using [git-cliff](ht
 
 ### Input variables
 
+- `version`: Git-cliff version to use (Default: `"latest"`)
```
Comment (orhun) [inline README.md:9]: nit:
```suggestion
- `version`: `git-cliff` version to use (Default: `"latest"`)
```

**Review by orhun — APPROVED** [review]
Thanks for your contribution!

I think I will go with `v4`, just to be safe :)

**Pull request merged**